releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,7 +1,16 @@
-## Microsoft.Extensions.Configuration.AzureAppConfiguration
-### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
+# Microsoft.Extensions.Configuration.AzureAppConfiguration
+[Source code][source_code] | [Package (NuGet)][package]
 
-### 7.0.0 - November 21, 2023
+## 7.1.0-preview - January 19, 2024
+### Enhancements
+* The provider will now automatically discover other existing replicas and attempt to connect to them when the AppConfig store and any replicas passed by the user have failed. This is enabled by default and can be disabled by setting the new `AzureAppConfigurationOptions.ReplicaDiscoveryEnabled` property equal to `false`. [#442](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/442)
```

Comment:
We shouldn't use "AppConfig" in official documentation. The "store and replicas" doesn't sound right. Replicas are part of a store. Suggest below:

For App Configuration stores with geo-replication enabled, the provider will automatically discover any additional replicas and attempt to connect to them when it fails to connect to user-provided endpoints in code. This capability allows applications to leverage geo-replication for enhanced resiliency without code change and redeployment.

Reply:
I like the summary, should I not mention the property?

Reply:
I skipped mentioning it because yours looks good. Sorry for the confusion. Yes, we should mention it. It's a new API.